ansible/test.yml

```diff
@@ -95,6 +95,21 @@
       that:
       - "sphinx_version.stdout == 'Sphinx (sphinx-build) 1.2.3'"
 
+- hosts: jekyll
+  roles:
+  - role: jekyll-site
+    jekyll_site_name: test
+    jekyll_site_git_repo: https://github.com/lunson/www.openmicroscopy.org
```

Comment:
That would work as well, we'll have to remember to update this test file eventually.